app/src/main/java/tftpserver/db/InMemDb.java

```diff
@@ -66,4 +66,10 @@ public List<byte[]> getFile(String filename) {
             lock.unlock();
         }
     }
+    @Override
+    public boolean isPresent(String filename) {
+        if(inMemDb.containsKey(filename)) {return true;}
+        return false;
+    }
```

Comment:
I think this method will go away, but in general, a fancy and short way of doing such operations are:
```java
public boolean isPresent(String filename) {
    return inMemDb.containsKey(filename)
}
```